src/components/Calendar.js

```diff
@@ -956,9 +887,8 @@ Calendar.propTypes = {
   ...FieldAdapterPropTypes,
   dayLimit: PropTypes.number.isRequired,
   forceRender: PropTypes.func,
-  updateTime: PropTypes.func,
   showAvailability: PropTypes.bool,
 }
 
-const CalendarAdapter = withContext(withI18n()(Calendar))
```

Comment:
Ok it's starting to look like this will overwrite all of our additions over the last couple weeks